sandbox/src/timebar/timebar.js

```diff
@@ -172,9 +202,55 @@ class TimebarContainer extends Component {
                 />
               )
             }
-            return <TimebarActivity key="activity" {...props} activity={activity} />
+            if (currentChart === 'track') {
+              return <>
+                <TimebarActivity
+                  {...props}
+                  key="trackActivity"
+                  color="#fe81eb"
```

Comment:
@nerik can't the color go within a css file?

Reply:
Probably this makes easier to have control of the color from the app that is using the component without issues of css specificity